--- CMakeLists_old.txt	2020-04-06 22:08:20.965358841 +0700
+++ CMakeLists.txt	2020-04-05 16:47:42.354422000 +0700
@@ -86,7 +86,7 @@
   SetVelocity.srv
   SetAttitude.srv
   SetRates.srv
-  SetLEDEffect.srv
+  #SetLEDEffect.srv
   Execute.srv
 )
 
@@ -172,7 +172,7 @@
 
 add_executable(vpe_publisher src/vpe_publisher.cpp)
 
-add_executable(led src/led.cpp)
+#add_executable(led src/led.cpp)
 
 add_executable(shell src/shell.cpp)
 
@@ -187,13 +187,13 @@
 
 target_link_libraries(vpe_publisher ${catkin_LIBRARIES})
 
-target_link_libraries(led ${catkin_LIBRARIES})
+#target_link_libraries(led ${catkin_LIBRARIES})
 
 target_link_libraries(shell ${catkin_LIBRARIES})
 
 add_dependencies(simple_offboard ${PROJECT_NAME}_generate_messages_cpp)
 
-add_dependencies(led ${PROJECT_NAME}_generate_messages_cpp)
+#add_dependencies(led ${PROJECT_NAME}_generate_messages_cpp)
 
 add_dependencies(shell ${PROJECT_NAME}_generate_messages_cpp)
 
@@ -277,4 +277,4 @@
 if (CATKIN_ENABLE_TESTING)
   find_package(rostest REQUIRED)
   add_rostest(test/basic.test)
-endif()
\ No newline at end of file
+endif()
